src/signals/incident/components/StepWizard/components/Step.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Step({ render }: { render: () => JSX.Element | null }) {
```

Comment:
> There are no tests for the newly added components (except for Wizard). Neither do I see something that would excluded the components from having to be tested. Am I missing something? How did you pass the threshold for coverage? Shouldn't we have tests since sofar every component seems to have a test.

imo its better to test this from the highest level, like in other parts in the codebase. Testing each every component is a good principle for components that are highly independent. But..here there actually the opposite and testing each file separately would result in duplication + time waste.